src/backend/config.ts

```diff
@@ -320,7 +320,9 @@ class GlobalConfigV0 extends GlobalConfig {
       },
       wineCrossoverBottle: 'Heroic',
       winePrefix: isWindows ? '' : defaultWinePrefix,
-      wineVersion: defaultWine
+      wineVersion: defaultWine,
+      enableEsync: true,
+      enableFsync: true
```

Comment:
this is changing the default to be `true` right? is this intentional?

I'm fine with that, just want to be sure this was the idea

Reply:
Yeah, I find it weird that we didn't have esync and fsync as default